src/add-triage-labels.ts

```diff
@@ -0,0 +1,44 @@
+import { Probot } from 'probot';
+import { DOCUMENTATION_LABEL, SEMANTIC_PREFIXES, SEMVER_LABELS, SEMVER_NONE_LABEL } from './constants';
+import { addLabels } from './utils/label-utils';
+
+export function addBasicPRLabels(probot: Probot) {
+  probot.on(
+    [
+      'pull_request.opened',
+      'pull_request.unlabeled',
+      'pull_request.labeled',
+      'pull_request.synchronize',
+    ],
+    async context => {
+      const { pull_request: pr } = context.payload;
+
+      const getSemanticPrefix = (title: string) => title.split(':')[0];
+      
+      // Label Docs PRs as Semver-Patch PRs.
+      const isDocsPR = getSemanticPrefix(pr.title) === SEMANTIC_PREFIXES.DOCS;
+      if (isDocsPR) {
+        await addLabels(context.octokit, {
+          ...context.repo({}),
+          prNumber: pr.number,
+          labels: [
+            SEMVER_LABELS.PATCH,
+            DOCUMENTATION_LABEL
+          ],
+        });
+      }
+
+      const isCIPR = getSemanticPrefix(pr.title) === SEMANTIC_PREFIXES.CI;
+      const isTestPR = getSemanticPrefix(pr.title) === SEMANTIC_PREFIXES.TEST;
+      const isBuildPR = getSemanticPrefix(pr.title) === SEMANTIC_PREFIXES.BUILD;
+
+      // CI, Test, and Build PRs do not affect Semver.
```

Comment:
this is _usually_ the case but not _necessarily_. e.g. a `build:` PR could change a GN build flag that affects the output, which would make it `semver/patch`.

Perhaps this should test for an existing `semver/*` label and respect that if it's already present?

Reply:
good point - i'll do that!

Reply:
probably same with the `docs` one!